### PR TITLE
fix: camelcase for options

### PR DESCRIPTION
**Pull request by danielbraun89**

### CODE REVIEW

File: src/pulumi/devcontainer-feature.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
             "default": "latest",
             "description": "Select the Pulumi version you would like to install"
         },
-        "bash_completion": {
+        "BashCompletion": {
```
Comment (eitsupi) [inline src/pulumi/devcontainer-feature.json:16]:
```suggestion
        "bashCompletion": {
```

Comment (danielbraun89) [reply src/pulumi/devcontainer-feature.json:16]: Didnt know how to reopen the PR so in this commit eee8eb0513bd4fa0920f9827da6f06cf959adcef

**Comment by Imihar** [comment]
Kue7o4orudtjjdoue6ieus ngeyneumeh.dmeo7ehmxmje7odb. N.eukueudumeo7e6khmhskue7osmuzh 6ieumdh.kye6kehd.he6ieumdmjdmhx
nmudmjdkue7lrhdhme7k3umf
Jjdku47oehd.hsmuei6wkye3iek7eo7ekyeku4l8t847jeyjw6ueh.sys6ke7kdmmduk3ieudumsuke6ismhdmuei737kumdudymeo7e7ld.hxymek7e7lrilrkuenuemek73k7ekudmudluso7e7odymztjs6kxv sykeyshetjsb syneynxmhdukeyneyksgmsnyeg si6s gw6jzgnengzjysntenysyk3iumur6ie gzhsnyd

**Comment by Imihar** [comment]
Lunxileouxb.s6oe7ldb ykeirbcb.e6i3ukxn.xum37kd  
jeooyshsmo6eudhsykeyksmhs6owkzhmskuwousjskuemujdk7dmhuo3lhdeuoeoydullurlurlueluekueluemurliekueummrkudludhmsyme6kdhchmsykdb
Culdykn
Syks.bxyneynxhmeymdum curye6e66ieiyo7e7oeynsh shmeyneekysluhxulsoue6ieujnxkyeulxbhmehrurkyd.bcbeiud.urul

**Comment by Imihar** [comment]
Mhzkys6ieymxn
xuoeujxlueludn
clurilfncilrljjclirifjdlujirlifjfluuur7ojchmsyneuffumeumrimriutidul3i7uljxmhukehncj

**Pull request merged**